list_to_email_converter.py

```diff
@@ -0,0 +1,234 @@
+import os
+from urllib.parse import quote
+import sys
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("path",type=str, help="the folderpath like: ./example/more/pytr")
+parser.add_argument("-s","--seek", help="seeks an existing entry", action="store_true")
+parser.add_argument("-m","--mailinglist", help="when you want to update the mailinglist", action="store_true")
+user_input = parser.parse_args()
+
+if user_input.seek:
+    if user_input.path.find('.md') == -1:
+        print("It is not a .md file!")
```

Comment:
Stop the program here (throw an error or just exit with an error message), because this is an input error.